marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs

```diff
@@ -182,7 +187,13 @@ runCreateCommand TxCommand { walletAddresses, signingMethod, tagsFile, metadataF
         case Map.toList configMap of
           [] -> throwE $ RolesConfigFileDecodingError "Empty role token config"
           (x : xs) -> pure $ RoleTokensMint $ mkMint $ fmap (\RoleConfig{..} -> (address, Just metadata)) <$> x :| xs
-    ContractId contractId <- run MarloweV1 minting'
+    (ContractId contractId, safetyErrors) <- run MarloweV1 minting'
+    liftIO
+      $ if null safetyErrors
+        then hPutStrLn stderr "Safety analysis found no errors in the contract."
+        else do
+               hPutStrLn stderr "Safety analysis found the following errors in the contract:"
+               BS8.hPutStrLn stderr $ Yaml.encode safetyErrors
```

Comment:
Safety errors are printed as YAML to `stderr`. Alternatively, we could have this written to a file.